sklearn/multioutput.py

```diff
@@ -217,6 +217,11 @@ class MultiOutputRegressor(MultiOutputEstimator, RegressorMixin):
         When individual estimators are fast to train or predict
         using `n_jobs>1` can result in slower performance due
         to the overhead of spawning processes.
+        
```

Comment:
Pep8 is complaining that you have added a blank line that contains white space. Blank lines must be empty.

Reply:
I don't understand, I believe it is complaining about the 3 spaces before the end of the docstring """,  but I see those 3 spaces in the docstring of other classes

Reply:
It should be fixed now